app/components/recognition/form.js

```diff
@@ -332,11 +319,45 @@ export default class FormComponent extends Component {
   }
 
   @action
-  handleAwardedByChange(governingBody) {
-    this.currentRecognition.selectedItem = governingBody;
+  handleAwardedByChange(organizationName) {
+    this.currentRecognition.selectedItem = organizationName;
+    this.clearFormError('awardedBy');
+  }
+
+  @action
+  handleAwardedByChangeEvent(event) {
+    this.currentRecognition.recognitionModel.awardedBy = event.target.value;
```

Comment:
I think we may want to trim the value so we're less likely to create a new one if the user added unneeded whitespace.

```suggestion
    this.currentRecognition.recognitionModel.awardedBy = event.target.value.trim();
```